src/reference/04-Howto/03-Howto-Generating-Files.md

```diff
@@ -103,6 +103,11 @@ resourceGenerators in Compile += Def.task {
 }.taskValue
 ```
 
+Executing `run` (or `package`) will add a file `demo` to `resourceManaged`,
+which is `target/scala-*/resource_managed"`. By default, generated resources are
+not included in the packaged source artifact. To do so, add them as you would
+other mappings. See [Adding files to a package][modify-package-contents].
```

Comment:
Can we add a small line here saying: "Note that running `compile` is not sufficient for `resourceGenerators` to run."